spring-cloud-generator/scripts/generate-steps.sh

```diff
@@ -35,8 +35,9 @@ function compute_monorepo_version() {
 # $1 - Monorepo version tag (or committish)
 function generate_libraries_list(){
   monorepo_commitish=$1
-  git clone --depth=1 https://github.com/googleapis/google-cloud-java.git
+  git clone --no-single-branch --depth=1 https://github.com/googleapis/google-cloud-java.git
   pushd google-cloud-java || { echo "Failure: google-cloud-java folder does not exists."; exit 1; }
+  git checkout "${MONOREPO_TAG}"
```

Comment:
How to setup this variable?

Reply:
It's explained in the main script. This is an environment variable that will have a default value if not set

https://github.com/GoogleCloudPlatform/spring-cloud-gcp/blob/7157231d8827809b82461c49ccbe9f74e6e4e108/spring-cloud-generator/scripts/generate.sh#L7-L12